examples/hostpath-provisioner/go.mod

```diff
@@ -3,14 +3,11 @@ module sigs.k8s.io/kubernetes-sigs/sig-storage-lib-external-provisioner/examples
 go 1.13
 
 require (
-	github.com/miekg/dns v1.1.27 // indirect
-	github.com/prometheus/client_golang v1.4.1 // indirect
-	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d // indirect
-	golang.org/x/time v0.0.0-20191024005414-555d28b269f0 // indirect
 	k8s.io/api v0.17.3
 	k8s.io/apimachinery v0.17.3
 	k8s.io/client-go v0.0.0-20200131194156-19522ff28802 // release-14.0
 	k8s.io/klog v1.0.0
-	k8s.io/utils v0.0.0-20200124190032-861946025e34 // indirect
-	sigs.k8s.io/sig-storage-lib-external-provisioner v4.0.1+incompatible
+	sigs.k8s.io/sig-storage-lib-external-provisioner/v5 v5.0.0
 )
+
+replace sigs.k8s.io/sig-storage-lib-external-provisioner/v5 => ../..
```

Comment:
Is this needed?

Reply:
Without it, "make" in `examples/hostpath-provisioner` will use the latest published release of `sigs.k8s.io/sig-storage-lib-external-provisioner/v5` instead of the current source code.

Now consider what happens when a breaking API change is made:
- first the current implementation of the lib gets updated
- a new release must be tagged which contains an example that still uses the old API
- only now can the example be updated